hooks/useNetAPR.ts

```diff
@@ -0,0 +1,291 @@
+import { useEffect, useMemo, useState } from 'react';
+import useAccountData from './useAccountData';
+import { WEI_PER_ETHER } from 'utils/const';
+import useDashboard from './useDashboard';
+import { parseEther } from 'viem';
+import { useWeb3 } from './useWeb3';
+import fetchAccounts, { Account } from 'queries/fetchAccounts';
+import useRewards from './useRewards';
+import useStETHNativeAPR from './useStETHNativeAPR';
+
+export default () => {
+  const [accounts, setAccounts] = useState<Account[]>([]);
+  const { walletAddress, subgraphURL } = useWeb3();
+  const { accountData = [] } = useAccountData();
+  const { floatingRows: floatingDeposit } = useDashboard('deposit');
+  const { floatingRows: floatingBorrow } = useDashboard('borrow');
+  const markets = useMemo(() => Object.values(accountData), [accountData]);
+  const { rates } = useRewards();
+  const stETHNativeAPR = useStETHNativeAPR();
+
+  useEffect(() => {
+    if (!subgraphURL || !walletAddress) return;
+    fetchAccounts(subgraphURL, walletAddress).then(setAccounts);
+  }, [subgraphURL, walletAddress]);
+
+  const fixedDepositsAPRs = useMemo(() => {
+    if (!accounts.length) return;
+    const _fixedPositions = accounts
+      .flatMap(({ market: { asset }, fixedPositions }) => fixedPositions.map((fp) => ({ ...fp, asset })))
+      .filter(({ borrow }) => !borrow);
+    return _fixedPositions.length
+      ? _fixedPositions.reduce(
+          (acc, { rate, asset, maturity }) => {
+            acc[asset.toLowerCase() + maturity] = rate;
```

Comment:
```suggestion
            acc[asset.toLowerCase() + maturity] = maturity < now ? 0n : rate;
```

Reply:
look exactly/lib https://github.com/exactly/lib/pull/2/files#:~:text=%7D)%20%3D%3E%20(maturity%20%3E%20timestamp

Reply:
you are right, thanks